CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## NEXT - UNRELEASED
+* Locatie van scg-icons.svg in variabele `$dso-icons-path` ondergebracht (default: `../`).
```

Comment:
Een changelog entry:
* valt onder een categorie. Zie de verwijzing bovenaan dit document naar Keep a Changelog. Ik denk dat dit een nieuwe feature is en daarmee onder de categorie "Added" valt;
* bevat een verwijzing naar de bijbehorende issue, zie alle andere changelog entries in deze file

Reply:
Aangepast.